src/components/app_catalog/app_list/index.js

```diff
@@ -44,23 +62,19 @@ class AppList extends React.Component {
   // filterApps is a pure function that takes a filter a list of all apps and returns
   // a filtered set of apps
   filterApps(allApps, filter) {
-    let apps = Array.from(Object.entries(allApps));
-    apps = apps.map(([, app]) => {
-      app.name = app[0].name;
-      app.version = app[0].version;
-      app.icon = app[0].icon;
-      return app;
-    });
+    console.log('f');
```

Comment:
This is me getting a feel how often this method gets called, was already doing some checking on the performance